tests/v2/validation/extensions/provisioning/providers/providers.go

```diff
@@ -12,8 +12,15 @@ import (
 	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials/linode"
 	"github.com/rancher/rancher/tests/framework/extensions/cloudcredentials/vsphere"
 	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
-	"github.com/rancher/rancher/tests/v2/validation/provisioning"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioning"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates"
+	r1aws "github.com/rancher/rancher/tests/framework/extensions/rke1/nodetemplates/aws"
```

Comment:
Can we have just the infrastructure as the name and not have `rke1` as a prefix to each of them?

Reply:
I agree with Markus if this is just RKE1 related plus I think this should be moved to extensions/providers in the framework